src/entry-point-client/App.tsx

```diff
@@ -1,37 +1,43 @@
+import React, { useState } from "react";
 import { hot } from "react-hot-loader";
 import { Route, Switch } from "react-router-dom";
 
 import AuthWall from "../auth/AuthWall";
 import { LocaleDataProvider } from "../locale-data-context";
+import { FacilityContext } from "../page-multi-facility/FacilityContext";
```

Comment:
Clearly this file shouldn't depend on something in the "page-multi-facility" package, but `FacilityContext` should also live in a more globally accessible package.